src/styles/themes.css

```diff
@@ -0,0 +1,15 @@
+.theme-classic {
```

Comment:
I'm using [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) and a _root_ CSS class  as a simple mechanism to support theming.

Probably, it would be a better idea switching to something more elaborated like [tailwindcss-theming plugin](https://github.com/hawezo/tailwindcss-theming).

Reply:
Making it possible for the admin of the site to define which theme to use is a nice feature. I think the approach you followed is enough for that.

I don't think that going further (dark mode, allowing the client to change the theme, select a theme based on the client device, etc.) is important enough to deserve the introduction of another not-so-small dependency.